scripts/inline-bootstrap-css.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
```

Comment:
This is so much more clear, comment def helps too. 👍

Reply:
It's not nice that we have to do it, but this is at least the most readable way possible 🙈